src/common/controllers/ads-controller.js

```diff
@@ -312,6 +316,26 @@ class AdsController extends FakeEventTarget implements IAdsController {
     });
   }
 
+  _handleLiveEvery(): void {
+    const liveConfigAdBreaks = [];
+    this._configAdBreaks.forEach(adBreak => {
+      const {every, ads} = adBreak;
+      if (this._player.duration && every) {
```

Comment:
```suggestion
      if (every && this._player.duration) {
```
Just to prevent unnecessary `this._player.duration` calls